### ISSUE TITLE
Some suggests  (awesome code)

### ISSUE THREAD

**Issue by mchobby** [ropod7/pyboard_drive#1]
Hi Ropod.

AWESOME WORK! Amazing!

Some suggestion:
- define a class (like) ILI9341_LCD to define the function.
  *\*  Allow to write lcd = ILI9341_LCD( spi_bus=1, csx = 'X4', dcx='X5', rst='X3', spi_speed=42000000 ). 
  *\* Allow to transform the function lcd_draw_rect() to lcd.rect()
- downgrading default speed to 25000000 (higher value cause lot of transmission issue with longer wire)

Thanks for your code sharing, it works really great.

**Comment by ropod7** [comment]
1. I wil define classes little bit later, I have done it as test code, as you see. Of course I have a plans for classes. It must be soon.
   Shore, it might optimize code and works faster in classes, less arguments, and logics = more speed. Somewhere we may realize functions with asm_thumb decorators. But I have not enough theory. :(
   I wish to know that better. I will find more information about assembler programming language.
   In micropython forum I am opened subject and guys gave me few solutions one of them was **asm_thumb** http://forum.micropython.org/viewtopic.php?f=6&t=1420
2. I do not understand about 25000000, if you mean SPI baudrate, it is unreal, because SPI 1 port generates rate only 42000000/2 = 21MHz (As I understood that feature.) Second port is less than 21MHz.

**Comment by mchobby** [comment]
About 2:
I was testing with longer wiring (+30cm). At baudrate=42000000 the communication bug the screen few after the screen reset (I did think about interference between cables as suggested on some SPI related posts). So I did shorten wire to about 15cm and start at very low rate (about 20000). It started to work properly so I increased the baudrate step by step.
I increased the baudate until 25000000 without issue... over that value communication is not always reliable.
Regarding the different test values for baudrate, I didn't take care about correct and incorrect value for the SPI Baudrate.
